packages/pageheader/stories/PageHeader.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withReadme } from "storybook-readme";
+
+import { PageHeader } from "../index";
+import { Clickable } from "../../clickable";
+
+const readme = require("../README.md");
+const action = () => alert("Action 1 triggered");
+class Example extends React.Component<{}, {}> {
+  render() {
+    return (
+      <PageHeader
+        breadcrumbElements={[
+          <span key="ElementOne">One</span>,
+          <span key="ElementTwo">Two</span>
+        ]}
+        actions={[
+          <Clickable key="Action1" action={action}>
+            <div>Action 1</div>
+          </Clickable>
+        ]}
+      />
+    );
+  }
+}
+
+storiesOf("PageHeader", module)
+  .addDecorator(withReadme([readme]))
+  .add("default", () => <Example />);
```

Comment:
Why aren't we just adding the JSX right in the `.add`?

Reply:
Because I missed replacing it, will adjust.